tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,228 @@
+import logging
+import time
+import json
+import pytest
+
+from ocsci import tier1, ManageTest
+import ocs.defaults as defaults
+import ocs.exceptions as ex
+from utility.utils import run_cmd
+from utility import templating
+from kubernetes import client, config
+from ocs.utils import create_oc_resource
+from ocs import (pod, ocp)
+
+log = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+# Project name
+PROJECT_NAME = "ocs-372"
+
+PRJ = ocp.OCP(kind='Project')
```

Comment:
do we have kind project?
I thought this is namespace

Reply:
In oc we use project.
@keesturam @nehaberry what is your thought?

Reply:
@Akarsha-rai @RazTamir - Yes. we have kind project. Project (oc object) is a wrapper around namespace (kube object). 
```
oc get project default -o yaml
apiVersion: project.openshift.io/v1
kind: Project <<------
metadata:
  annotations:
    openshift.io/sa.scc.mcs: s0:c11,c5
    openshift.io/sa.scc.supplemental-groups: 1000120000/10000
    openshift.io/sa.scc.uid-range: 1000120000/10000
  creationTimestamp: "2019-06-01T20:02:37Z"
  name: default
  resourceVersion: "7971"
  selfLink: /apis/project.openshift.io/v1/projects/default
  uid: 3461be75-84a8-11e9-9c05-02e38db053da
spec:
  finalizers:
  - kubernetes
status:
  phase: Active
```